public/css/main.css

```diff
@@ -562,25 +563,186 @@ hr {
   height: 0;
   margin-bottom: 4rem; }
 
+.footer {
+  background: var(--color-accent1);
+  padding: var(--margin-s);
+  margin: 0; }
+  .footer.row {
+    margin-bottom: 0; }
+
+@media screen and (min-width: 750px) {
+  .footer {
+    padding: var(--margin-m); } }
+
+@media screen and (min-width: 1000px) {
+  .footer {
+    padding: var(--margin-m) var(--margin-l); } }
+
+@media only screen and (min-width: 1200px) {
+  .footer {
+    padding: var(--margin-m) var(--margin-xl); } }
+
 p, ol, ul, dl {
   margin: 0 0 var(--margin-s) 0; }
 
-.container, .container-small, .container-medium, .container-large, .container-space-between {
-  padding: var(--padding-m);
-  margin: 0 auto; }
-
 .container {
   overflow-x: auto;
-  margin-bottom: var(--margin-m);
-  padding: 0; }
-  .container-small {
-    max-width: var(--container-small); }
-  .container-medium {
-    max-width: var(--container-medium); }
-  .container-large {
-    max-width: var(--container-large); }
-  .container-space-between {
-    display: flex;
-    flex-direction: row;
-    justify-content: space-between;
-    flex-wrap: wrap; }
+  padding: var(--padding-m);
+  margin: 0 var(--margin-s); }
+
+@media screen and (min-width: 750px) {
+  .container {
+    margin: 0 var(--margin-m); } }
+
+@media screen and (min-width: 1000px) {
+  .container {
+    margin: 0 var(--margin-l); } }
+
+@media only screen and (min-width: 1200px) {
+  .container {
+    margin: 0 var(--margin-xl); } }
+
+.max-width-small {
```

Comment:
This is not working as I want it to. elements do not keep the minimal width of the grid. 
Have to look into another way of centering items in the grid.

Reply:
removed it and replaced it with explicit grid declarations for centering